tutorials/applications/napari_imageJ.md

```diff
@@ -20,26 +20,26 @@ except ImportError:
     raise ImportError("""This example uses ImageJ but pyimagej is not
     installed. To install try 'conda install pyimagej'.""")
 
-with napari.gui_qt():
 
-    print('--> Initializing imagej')
-    ij = imagej.init('sc.fiji:fiji') # Fiji includes Bio-Formats.
+print('--> Initializing imagej')
+ij = imagej.init('sc.fiji:fiji') # Fiji includes Bio-Formats.
 
-    for i in range(1, len(sys.argv)):
-        path = sys.argv[i]
-        print('--> Reading {}'.format(path))
+for i in range(1, len(sys.argv)):
+    path = sys.argv[i]
+    print('--> Reading {}'.format(path))
 
-        dataset = ij.io().open(path)
-        image = ij.py.from_java(dataset)
+    dataset = ij.io().open(path)
+    image = ij.py.from_java(dataset)
 
-        if i == 1:
-            viewer = napari.view_image(image)
-        else:
-            viewer.add_image(image)
+    if i == 1:
+        viewer = napari.view_image(image)
+    else:
+        viewer.add_image(image)
 
-    ij.getContext().dispose()
+ij.getContext().dispose()
 
-    viewer.grid_view()
+viewer.grid_view()
+napari.run()
```

Comment:
Shouldn't napari already be initialized/running by now? What have I missed that makes this line necessary?

Reply:
Oh hang on, it seems like some of our examples are being run as if they were notebook cells, and some are being run as scripts, which I missed while skimming. I'd prefer more consistency. When are we choosing one format over the other?

Reply:
this one was externally contributed, so I wouldn't say "we" have chosen either format over the other yet.  It's just not a conversation that's been had yet...

> What have I missed that makes this line necessary?

the general rule is this: if you're not in ipython, you have to call `napari.run()` to use napari.  Each time `napari.run()` appears in a script, it will block the script and control goes to the viewer.  If you're in IPython, it's not necessary (but it also isn't harmful, so if you want something to work as a script and work in IPython `%run`, then use `napari.run()` like you would `plt.show()`)

Reply:
We should state the information in your last paragraph in the docs